src/reducers/contacts.reducer.spec.js

```diff
@@ -0,0 +1,137 @@
+import * as actions from '../actions';
```

Comment:
using import * is not the best way to import things. best thing you can do is group all the actions from types into a class and import that class here, and everywhere it is needed